ironfish-cli/src/commands/chain/genesisadd.ts

```diff
@@ -0,0 +1,203 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { MEMO_LENGTH } from '@ironfish/rust-nodejs'
+import {
+  addGenesisTransaction,
+  BlockSerde,
+  CurrencyUtils,
+  GenesisBlockAllocation,
+  IJSON,
+  isValidPublicAddress,
+} from '@ironfish/sdk'
+import { CliUx, Flags } from '@oclif/core'
+import fs from 'fs/promises'
+import { IronfishCommand } from '../../command'
+import { LocalFlags } from '../../flags'
+
+export default class GenesisAddCommand extends IronfishCommand {
+  static hidden = true
+
+  static flags = {
+    ...LocalFlags,
+    account: Flags.string({
+      char: 'a',
+      required: true,
+      description: 'The name of the account to use for keys to assign the genesis block to',
+    }),
+    allocations: Flags.string({
+      required: true,
+      description:
+        'A CSV file with the format address,amountInIron,memo containing genesis block allocations',
+    }),
+    expectedSupplyInIron: Flags.string({
+      char: 'g',
+      required: true,
+      description: 'The amount of coins in the genesis block',
```

Comment:
This a bit more like: 
```suggestion
      description: 'The total prior allocation to the given account',
```